### DISCUSSION TITLE
bug(Multiselect): 雙向綁定畫面異常

### DISCUSSION THREAD

**Discussion by a0926019412** [dotnetcore/BootstrapBlazor#3540] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

<img width="311" alt="image" src="https://github.com/dotnetcore/BootstrapBlazor/assets/4286244/1c46908b-6437-4a36-af2c-f084985c9a6d">

如附圖, 之前選了勾勾以後會在textbox中顯示已選擇的標籤, 現在不顯示了(只有雙向綁定才有這個異常)
但值有綁定到

### .NET Version

6.0

**Comment by bb-auto** [comment]
@a0926019412 Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Reply by ArgoZhang** [reply, marked as answer]
@a0926019412 网站上我都做了例子。可以去看看

https://www.blazor.zone/table/edit#EditTemplate

**Reply by a0926019412** [reply]
問題同 #3491
